en/advanced-topics/dynamic-atlas.md

```diff
@@ -0,0 +1,37 @@
+# Dynamic Atlas
+
+Reducing the number of DrawCalls is a very direct and effective way to improve game rendering efficiency, and one of the most important factors for two DrawCalls to be merged into one is whether both DrawCalls use the same texture.
```

Comment:
, and 
-->
, and is

Reply:
Actually "one of the most important factors for..." is another sentence. Maybe I should delete ", and" and add a period?

Reply:
```suggestion
Reducing the number of DrawCalls is a very direct and effective way to improve game rendering efficiency. One of the most important factors for two DrawCalls to be merged into one is whether both DrawCalls use the same texture.
```